src/main/kotlin/com/ajaxproject/telegrambot/bot/BankInfoBot.kt

```diff
@@ -1,36 +1,49 @@
 package com.ajaxproject.telegrambot.bot
 
-import com.ajaxproject.telegrambot.bot.commands.AbstractCommand
+import com.ajaxproject.telegrambot.bot.model.UserRequest
+import com.ajaxproject.telegrambot.bot.service.UserSessionService
 import org.slf4j.LoggerFactory
-import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Component
 import org.telegram.telegrambots.bots.TelegramLongPollingBot
 import org.telegram.telegrambots.meta.api.objects.Update
-
 @Component
 class BankInfoBot(
-    @Qualifier("commandsMap")
-    private val commandsMapping: Map<String, AbstractCommand>,
+    val dispatcher: Dispatcher,
+    val userSessionService: UserSessionService,
 ) : TelegramLongPollingBot() {
-
-    @Value("\${credentials.botUsername}")
+    @Value("\${bot.username}")
     private val botUsername: String = ""
 
-    @Value("\${credentials.botToken}")
+    @Value("\${bot.token}")
     private val token: String = ""
+    override fun getBotToken(): String = token
 
     override fun getBotUsername(): String = botUsername
-    override fun getBotToken(): String = token
 
     override fun onUpdateReceived(update: Update) {
-        if (update.hasMessage()) {
-            val text = update.message.text
-            commandsMapping[text]?.handle(update, this)
-        } else if (update.hasCallbackQuery()) {
-            val callBack = update.callbackQuery.data
-            log.debug("New request with callbackQuery: {}", callBack)
-            commandsMapping[callBack]?.handle(update, this)
+        if (update.hasMessage() && update.message.hasText()) {
+            val textFromUser = update.message.text
+
+            val userId = update.message.from.id
+            val userFirstName = update.message.from.firstName
+
+            log.info("[{}, {}] : {}", userId, userFirstName, textFromUser)
```

Comment:
it's better to add more context to the log message.

Something like 'User {} {} wrote text {}'.

Actually, there are two issues:
- it looks like debug message, so it's better to use debug log level instead of info
- it looks insecure. You log user's message... If it wasn't the pet project I wouldn't allowed to do this